ddp-workspace/projects/ddp-singular/src/assets/i18n/en.json

```diff
@@ -93,7 +93,7 @@
           {
             "Name": "How will Project Singular learn about single ventricle heart defects?",
             "Answers": [
-              "We will collect a saliva (spit) sample or a cheek swab from people enrolled in this study. We will use these samples to look at DNA. We want to see if we can find the causes of single ventricle heart defects. If we understand the causes, researchers may be able to develop new and better treatments.",
+              "We will collect a saliva sample or a cheek swab from people enrolled in this study. We will use these samples to look at DNA. We want to see if we can find the causes of single ventricle heart defects. If we understand the causes, researchers may be able to develop new and better treatments.",
```

Comment:
It's written in the ticket "take out the mention of cheek swab".
Thus, the first sentense should be 
`We will collect a saliva sample from people enrolled in this study.`